docs/src/components/DocsLayout.tsx

```diff
@@ -67,38 +67,75 @@ class DocsLayout extends React.Component<any, any> {
     })
   }
 
-  renderChildren = props => {
-    const { component: Children, render } = this.props
+  renderChildren() {
+    const { children, render } = this.props
+    const sidebarWidth = '270px'
+
+    const treeSectionStyle = {
+      fontWeight: fontWeightBold,
```

Comment:
```suggestion
      fontWeight: 700,
```

It can be safer to inline it for now because other values are hardcoded now